File: typescript/docs_code_snippets/spans-delete.ts
import { Opper } from "opperai";

const opper = new Opper();

// setup
const created = await opper.spans.create({ name: "docs-snippet-span" });
const spanId = created.id;
// /setup

// --- docs ---
await opper.spans.deleteSpan(spanId);
// --- /docs ---

console.log("Span deleted");

File: python/docs_code_snippets/spans-delete.py
from datetime import datetime, timezone
from opperai import Opper

opper = Opper()

# Setup
span = opper.spans.create(
    name="my-pipeline",
    start_time=datetime.now(timezone.utc).isoformat(),
)

# --- docs ---
opper.spans.delete(span.id)
# --- /docs ---
